src/Build/Evaluation/Profiler/EvaluationLocationPrettyPrinterBase.cs

```diff
@@ -75,7 +75,7 @@ protected static string GetElementOrConditionText(string description, Evaluation
         protected void AppendDefaultHeaderWithSeparator(StringBuilder stringBuilder, string separator)
         {
             stringBuilder.AppendLine(
-                string.Join(separator, ["Id", "ParentId", "Pass", "File", "Line #", "Expression", "Inc (ms)", "Inc (%)", "Exc (ms)",
+                string.Join(separator, (string[])["Id", "ParentId", "Pass", "File", "Line #", "Expression", "Inc (ms)", "Inc (%)", "Exc (ms)",
```

Comment:
AFAICT, the better fix is to remove the `[` and `]`, then it becomes an expanded params call to either string[] or `ReadOnlySpan<string>`, depending on the TFM you're building against.

If for some reason you're building against a new TFM but hoping that it magically works against an older one, this does force it to go to the older codepath (and allocate the array in order to do so)